contracts/entrypoint.sh

```diff
@@ -31,10 +31,39 @@ if [ "${DEPLOY_TYPE}" = "core" ]; then
         --keystores "${KEYSTORE_DIR}/${KEYSTORE_FILENAME}" \
         --password "${KEYSTORE_PASSWORD}" \
         --sender "${SENDER}" \
-        --skip-simulation \
         --use 0.8.26 \
         --broadcast \
-        --force \
         --json \
         --via-ir
-fi 
+elif [ "${DEPLOY_TYPE}" = "settlement-gateway" ]; then
+    echo "Deploying gateway contract on settlement chain"
+    "${FORGE_BIN_PATH}" script \
+        "${SCRIPT_PATH_PREFIX}/standard-bridge/DeployStandardBridge.s.sol:DeploySettlementGateway" \
+        --root "${CONTRACT_REPO_ROOT_PATH}" \
+        --priority-gas-price 2000000000 \
```

Comment:
Not a big deal but I'd bet the newer versions of foundry don't need explicit gas params to be set anymore

Reply:
Unfortunately, without it, the contract deployment hangs forever.